docs/index.rst

```diff
@@ -10,13 +10,53 @@ eProsima Micro XRCE-DDS
 This implementation complies with the specification proposal, "eXtremely Resource Constrained Environments DDS (DDS-XRCE)" submitted to the Object Management Group (OMG) consortium.
 
 *Micro XRCE-DDS* implements a client-server protocol to enable resource-constrained devices (clients) to take part in DDS communications.
-*Micro XRCE-DDS Agent* (server) makes possible this communication.
-The *Micro XRCE-DDS Agent* acts on behalf of the *Micro XRCE-DDS Clients* and enables them to take part as DDS publishers and/or subscribers in the DDS Global Data Space.
+*Micro XRCE-DDS Agent* (server) makes possible this communication acting on behalf of the *Micro XRCE-DDS Clients*
+and enables them to take part as DDS publishers and/or subscribers in the DDS Global Data Space.
 
 *Micro XRCE-DDS* provides both, a plug and play *Micro XRCE-DDS Agent* and an API layer which allows you to implement your *Micro XRCE-DDS Clients*.
 
 .. image:: images/xrcedds_architecture.svg
 
+Main features
+~~~~~~~~~~~~~
+High performance.
+    Uses a static low level serialization library `(MicroCDR) <https://github.com/eProsima/Micro-CDR>`_ that serialize in `XCDR <https://www.omg.org/spec/DDS-XTypes/1.2/PDF>`_.
+
+Low resouces.
+    The client library is dynamic memory free and static memory free.
+    This means that the only memory charge is due by the stack growth.
+    It can manage a simple publisher/subscriber with less of 2KB of RAM.
+    Besides, the client is built with a *profiles* concept, that allows to add or remove functionality to the library in order to reduce the binary size.
+
+Multi-platform.
+    The OS dependencies are treated as pluggable modules.
+    The user can implement easily his own platform modules in order to use *Micro XRCE-DDS Client* library in his specific platform.
+    By default the proyect can run over `Linux`, `Windows` and `Nuttx`.
+
+Free and Open Source.
+   The client library, among with the agent executable, the generator tool and internal dependencies as *MicroCDR* or *FastRTPS* are all of them free and open source.
+
+Easy to use.
+    The project comes with examples of a publisher and a subscriber,
+    an example of how MicroXRCE-DDS is deployed into a stage and
```

Comment:
```suggestion
    an example of how Micro XRCE-DDS is deployed into a stage and
```

Reply:
I am not sure what do you mean with `stage`.

Reply:
Into a "real scenario"